provisioning/miniprov/hare_mp/main.py

```diff
@@ -354,6 +361,56 @@ def start(args):
         start_hax_and_consul_without_systemd(url, utils)
 
 
+def start_mkfs(utils: Utils, hostname: str, hare_config_dir: str):
+    # TODO: path needs to be updated according to the new conf-store key
+    motr_config_dir = '/etc/motr'
```

Comment:
Let's fix this before we merge it as the mini provisioner changes are in. Can you please sync up with @SwapnilGaonkar7 ?

Reply:
I think this is working so, resolving this.